mu4/notation/qml/MuseScore/NotationScene/ViewModeControl.qml

```diff
@@ -0,0 +1,31 @@
+import QtQuick 2.7
+import MuseScore.NotationScene 1.0
+import MuseScore.UiComponents 1.0
+import MuseScore.Ui 1.0
+
+Row {
+
+    ViewModeControlModel {
+        id: model
+    }
+
+    Component.onCompleted: {
+        model.load()
+    }
+
+    StyledComboBox {
+
+        
+        id: viewModeBox
+
+        opensUpward: true
+        maxVisibleItemCount: 3
+
+        textRoleName: "nameRole"
+        valueRoleName: "idRole"
+
+        model: model
```

Comment:
I'd suggest to make StyledComboBox to be a "root" item and declare ViewModeControlModel at this line